app/src/main/res/layout/details_fragment.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
```

Comment:
I am trying to use mostly constraint layouts since they are better to be processed. However depending on the complexity or objective of the screen a FrameLayout or LinearLayout could prove to be useful like in the landscape layout where a simple ScrollView was used with a LinearLayout. The only layout I don't think that has advantages against the ConstraintLayout would be the RelativeLayout since they work similarly.